packages/skygear-web/src/container.ts

```diff
@@ -1,29 +1,76 @@
 /* global window */
 import {
-  Container,
   User,
   ContainerOptions,
   GlobalJSONContainerStorage,
-  OIDCContainer,
+  BaseContainer,
   AuthorizeOptions,
 } from "@skygear/core";
 import { WebAPIClient } from "./client";
 import { localStorageStorageDriver } from "./storage";
 import { generateCodeVerifier, computeCodeChallenge } from "./pkce";
 
 /**
- * Skygear OIDC APIs (for web platforms).
- *
  * @public
  */
-export class WebOIDCContainer<T extends WebAPIClient> extends OIDCContainer<T> {
+export interface ConfigureOptions {
+  /**
+   * The OAuth client ID.
+   */
   clientID: string;
-  isThirdParty: boolean;
+  /**
+   * The endpoint.
+   */
+  endpoint: string;
+  /**
+   * isThirdPartyApp indicate if the application a third party app.
+   * A third party app means the app doesn't share common-domain with Skygear Auth thus the session cookie cannot be shared.
```

Comment:
Here need fix name.

Reply:
Let's fix it in another PR, including renaming the package names, replacing occurrence of skygear, etc.